docs/source/index.rst

```diff
@@ -51,7 +51,7 @@ Dataset evaluation provide tools to train and evaluate ML models for different d
       --config=datasetinsights/configs/faster_rcnn.yaml \
       --test-data=$HOME/data
 
-To learn more, see this :doc:`Evaluation_Tutorial` Documentation.
+To learn more, see this `tutorial <https://datasetinsights.readthedocs.io/en/latest/Evaluation_Tutorial.html>`_.
```

Comment:
Please revert this. ":doc:`Evaluation_Tutorial`" is actually a valid sphinx link to  documents.